content/docs/configuration/traffic-management/transformations.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Transformations
+weight: 10
+description: Modify header and body information for requests and responses. 
+---
+
+Agentgateway uses transformation templates that are written in Common Expression Language (CEL). CEL is a fast, portable, and safely executable language that goes beyond declarative configurations. CEL lets you develop more complex expressions in a readable, developer-friendly syntax.
+
+To learn more about how to use CEL, refer to the following resources:
+
+- [cel.dev tutorial](https://cel.dev/tutorials/cel-get-started-tutorial)
+- [Agentgateway reference docs](https://agentgateway.dev/docs/reference/cel/)
+
+{{< callout >}}
```

Comment:
I am not sure if this is intentional...without the `type` attribute, it shows up as orange with no icon:
<img width="886" height="119" alt="image" src="https://github.com/user-attachments/assets/119dcbda-dc16-44b0-b191-8b854995be70" />

with the type=info, it looks like this:
<img width="876" height="130" alt="image" src="https://github.com/user-attachments/assets/22b11d20-5ab9-43d7-8186-a541944234da" />
```suggestion
{{< callout type="info">}}
```